src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -540,16 +540,16 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                     ->thenInvalid('"supports" or "support_strategy" should be configured.')
                                 ->end()
                                 ->beforeNormalization()
-                                        ->always()
```

Comment:
Even if I agree with the change, please revert: it's not related to the PR and if you changed nothing else in this file